app/models/project.rb

```diff
@@ -374,4 +377,8 @@ def has_minimum_goal_for_video?
   def reject_project_image(attributes)
     new_record? && attributes[:original_image_url].nil?
   end
+
+  def can_has_associated_plans?
```

Comment:
What do you mean by "another example" :smile: ? You can check the ruby style guide.
https://github.com/bbatsov/ruby-style-guide/blob/master/README.md#naming